### PR TITLE
Adding testimonials about the True God

### PR DESCRIPTION
**Pull request by naurojunior**
Praise the lord!

### CODE REVIEW

**Comment by lsunsi** [comment]
Praised be!
Can we get a picture of how the page is after the PR?

The class names gave me chills. 10/10 would praise again

File: testimonials.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      background-image: url(./god.jpg);
+      background-position: center;
+      background-size: cover;
+      height: 100vh;
+      width: 100vw;
+      text-align: center;
+    }
+
+    .🗣️{
+    	float: left;
```
Comment (AgtLucas) [inline testimonials.html]: Do not use float, please.

Otherwise it will curse you with McDonald's and BK.

File: testimonials.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    	margin: auto;
+    	width: 60vw;
+    	left: 0;
+		right: 0;
+		text-align: center;
+		position: absolute;
```
Comment (AgtLucas) [inline testimonials.html]: Can you please fix the indentation?

File: testimonials.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    .🗣️🍔{
+    	font-style: italic;
+    	background-color: white;
+    	padding: 15px;
+    	border: 1px solid #000;
+    	border-radius: 3px;
+		margin-top: 50px;
```
Comment (AgtLucas) [inline testimonials.html]: Same here.

File: testimonials.html (hunk 1 above)
Comment (naurojunior) [reply testimonials.html]: Sorry, the lack of Blue Velvet (about 2 weeks without one) made me do horrible stuff, like using float. I'm not in my best mental health conditions.

Praise the lord.

**Comment by naurojunior** [comment]
As asked by @lsunsi , the print of the page of testimonials.
Maybe we should add some link to this page.

Sorry for the bunch of commits, Sublime is really f*cking me up with identation. I must fix this, but I must eat THE LORD before, to regain mental health.

image

**Review by lsunsi — APPROVED** [review]
I'M GOOD WITH THIS
PRAISE THE BIG BLUE

(:shipit: after agent lucas approve)

**Review by AgtLucas — APPROVED** [review]
LGTM.

**Pull request merged**